src/lib.rs

```diff
@@ -190,6 +200,22 @@ pub trait ProcessHandleExt {
     fn set_arch(self, arch: Architecture) -> Self;
 }
 
+/// Handling modules (e.g. DLLs) in a process.
+pub trait ModuleInfo {
+    /// Gets the base address of a module in a process. For example, "GameAssembly.dll" when on Windows.
```

Comment:
The equivalent for dlls on Windows on Linux and OSX are "Shared Libraries" (which usually have the extension `.so`). Would you mind also mentioning that in docs?